connexion/decorators/security.py

```diff
@@ -252,7 +252,7 @@ def verify_apikey(apikey_info_func, loc, name):
 
     def wrapper(request, required_scopes):
```

Comment:
I would nest `_immutable_pop()` within `def wrapper` because it's only used here.
The reason to keep it as a function (rather than inlining) is just to clearly isolate and define what we're trying to do, and give the opportunity to include the docstring.